### PR TITLE
update cleanUpImages

### PR DESCRIPTION
**Pull request by eriksywu**

### CODE REVIEW

File: parts/linux/cloud-init/artifacts/cse_install.sh [hunk 1]
```diff
@@ -340,7 +340,6 @@ removeContainerImage() {
 cleanUpImages() {
     local targetImage=$1
     export targetImage
-    export -f removeContainerImage
```
Comment (bowang-666) [inline parts/linux/cloud-init/artifacts/cse_install.sh:343]: looks like removeContainerImage is needed here since it'll be used in retrycmd_if_failure?

Comment (eriksywu) [reply parts/linux/cloud-init/artifacts/cse_install.sh:343]: I've moved it to cleanUpContainerImages where all the other exports are for clarity...testing if that works

File: parts/linux/cloud-init/artifacts/cse_install.sh [hunk 2]
```diff
@@ -351,7 +350,8 @@ cleanUpImages() {
         if [[ $exit_code != 0 ]]; then
             exit $exit_code
         elif [[ "${images_to_delete}" != "" ]]; then
-            for image in "${images_to_delete[@]}"
+            images=$(${images_to_delete}) #alternatively use IFS+read but we are using bash anyways so this works too
```
Comment (bowang-666) [inline parts/linux/cloud-init/artifacts/cse_install.sh:353]: I believe you are looking for images=(${images_to_delete})
some local tests:
root@aks-nodepool1-34284155-vmss000006:/#             images=(${images_to_delete})
root@aks-nodepool1-34284155-vmss000006:/#             for image in "${images[@]}"
>             do
>                 echo ${image}
>             done
mcr.microsoft.com/oss/kubernetes/hyperkube:v1.18.10-hotfix.20210118
mcr.microsoft.com/oss/kubernetes/hyperkube:v1.17.16
mcr.microsoft.com/oss/kubernetes/hyperkube:v1.17.13
mcr.microsoft.com/oss/kubernetes/hyperkube:v1.17.11-hotfix.20200901.1
mcr.microsoft.com/oss/kubernetes/hyperkube:v1.18.8-hotfix.20200924
mcr.microsoft.com/oss/kubernetes/hyperkube:v1.16.15-hotfix.20200903
mcr.microsoft.com/oss/kubernetes/hyperkube:v1.17.11-hotfix.20200901
mcr.microsoft.com/oss/kubernetes/hyperkube:v1.17.9-hotfix.20200824.1
mcr.microsoft.com/oss/kubernetes/hyperkube:v1.16.10-hotfix.20200824.1
mcr.microsoft.com/oss/kubernetes/hyperkube:v1.16.13-hotfix.20200824.1
mcr.microsoft.com/oss/kubernetes/hyperkube:v1.17.7-hotfix.20200714.2
mcr.microsoft.com/oss/kubernetes/hyperkube:v1.18.6-hotfix.20200723.1
mcr.microsoft.com/oss/kubernetes/hyperkube:v1.18.4-hotfix.20200626.1
mcr.microsoft.com/oss/kubernetes/hyperkube:v1.17.3-hotfix.20200601.1
mcr.microsoft.com/oss/kubernetes/hyperkube:v1.16.9-hotfix.20200529.1
root@aks-nodepool1-34284155-vmss000006:/#

File: parts/linux/cloud-init/artifacts/cse_install.sh (hunk 1 above)
Comment (bowang-666) [reply parts/linux/cloud-init/artifacts/cse_install.sh:343]: right that would export for cleanUpImages. but I believe when cleanUpImages calls retrycmd_if_failure, it has to be exported again. lmk if that's not true.

Comment (xuto2) [reply parts/linux/cloud-init/artifacts/cse_install.sh:343]: I tested locally, cleanUpImages runs after the export so it still works.

**Pull request merged**